### ISSUE TITLE
test_conch, test_cftp and test_keys fail to import if Crypto.Cipher is not installed

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11179]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| spiv reported|
|-|-|
|Trac ID|trac#1674|
|Type|defect|
|Created|2006-04-27 16:51:06Z|

See http://twistedmatrix.com/buildbot/OS-X/builds/898/step-default/2:

```
===============================================================================
[ERROR]: twisted.conch.test.test_cftp

  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/python/reflect.py", line 352, in namedAny
    topLevelPackage = __import__(trialname)
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/test/test_cftp.py", line 31, in ?
    import test_ssh, test_conch
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/test/test_conch.py", line 123, in ?
    from test_keys import publicRSA_openssh, privateRSA_openssh
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/test/test_keys.py", line 10, in ?
    from twisted.conch.ssh import keys
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/ssh/keys.py", line 19, in ?
    from Crypto.Cipher import DES3
exceptions.ImportError: No module named Crypto.Cipher
===============================================================================
[ERROR]: twisted.conch.test.test_conch

  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/python/reflect.py", line 352, in namedAny
    topLevelPackage = __import__(trialname)
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/test/test_conch.py", line 123, in ?
    from test_keys import publicRSA_openssh, privateRSA_openssh
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/test/test_keys.py", line 10, in ?
    from twisted.conch.ssh import keys
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/ssh/keys.py", line 19, in ?
    from Crypto.Cipher import DES3
exceptions.ImportError: No module named Crypto.Cipher
===============================================================================
[ERROR]: twisted.conch.test.test_keys

  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/python/reflect.py", line 352, in namedAny
    topLevelPackage = __import__(trialname)
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/test/test_keys.py", line 10, in ?
    from twisted.conch.ssh import keys
  File "/Users/buildbot/slaves/twisted/OSX-full2.4/Twisted/twisted/conch/ssh/keys.py", line 19, in ?
    from Crypto.Cipher import DES3
exceptions.ImportError: No module named Crypto.Cipher
===============================================================================
```

These tests should instead be skipped gracefully.

<details><summary>Searchable metadata</summary>

```
trac-id__1674 1674
type__defect defect
reporter__spiv spiv
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1146156666000000 1146156666000000
changetime__1222181772000000 1222181772000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

Fixed as part of [#3111](https://github.com/twisted/twisted/issues/3111).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
